pylintrc

```diff
@@ -28,4 +28,5 @@ disable=bad-continuation,
         unused-argument,
         use-a-generator,
         wrong-import-order,
-        wrong-import-position
+        wrong-import-position,
+        unused-variable
```

Comment:
Why is this needed now? [The CI run in the main branch 5 minutes ago and pylint checks passed](https://github.com/scrapy/parsel/runs/5813996395?check_suite_focus=true).

Reply:
For which line did pylint generate that error?

Reply:
This is the error message that I am getting when I run tox -e pylint

![Screenshot (24)](https://user-images.githubusercontent.com/52698821/161876195-19eb4166-3837-4ccd-869d-fdbbcc910b59.png)

Reply:
`pylintrc` has `ignore=tests/typing`, maybe that's not enough in some cases?

Reply:
I think we may need to change that to `ignore=typing`.

From [the documentation of --ignore](https://pylint.pycqa.org/en/latest/user_guide/run.html):

> Files or directories to be skipped. **They should be base names, not paths.**

Reply:
After running tox with `ignore=typing`, I did not get any errors. I'll push the changes now